stress/shell/src/main/java/alluxio/stress/cli/StressMasterBench.java

```diff
@@ -172,14 +179,13 @@ public void prepare() throws Exception {
 
     hdfsConf.set(PropertyKey.Name.USER_FILE_WRITE_TYPE_DEFAULT, mParameters.mWriteType);
 
+    LOG.info("Using {} to perform the test.", mParameters.mClientType);
     if (mParameters.mClientType == FileSystemClientType.ALLUXIO_HDFS) {
-      LOG.info("Using ALLUXIO HDFS Compatible API to perform the test.");
       mCachedFs = new FileSystem[mParameters.mClients];
       for (int i = 0; i < mCachedFs.length; i++) {
         mCachedFs[i] = FileSystem.get(new URI(mParameters.mBasePath), hdfsConf);
       }
-    } else {
-      LOG.info("Using ALLUXIO Native API to perform the test.");
+    } else if (mParameters.mClientType == FileSystemClientType.ALLUXIO_NATIVE) {
```

Comment:
What about if the clientType is other type than the two?

Reply:
other client types e.g. Posix  API does not need the prepare stage